Framework/Algorithms/src/FilterEvents.cpp

```diff
@@ -108,9 +108,13 @@ void FilterEvents::init() {
                   "workspaces.  Group name will be "
                   "OutputWorkspaceBaseName.");
 
+  declareProperty("DescriptiveOutputNames", false,
```

Comment:
We need to be slightly careful when adding new properties to existing algorithms. The Python functions that we generate for users to run take the properties in the order that they are declared in the algorithm. In this case if a user calls the `FilterEvents` function in Python with positional parameters only then its possible they thought they were setting `OutputWorkspaceIndexedFrom1` before but would now set `DescriptiveOutputNames`. Could we move the new property to the end of the list?